doc/src/examples/access-control.md

```diff
@@ -88,43 +74,39 @@ impl PSP721Struct {
 }
 ```
 
-7. Let's customize it. We will implement `IPSP721Mint` trait. For that we need inherit `PSP721Mint`. It will
-   call `only_minter` function inside to verify that caller has minter role. Also, we need to update constructor to
-   grant minter role to caller by default.
+6. Customize it by adding access control logic. We will implement `IPSP721Mint` trait. It will use modifier `only_minter`(it verifies that caller
+   has minter role). Also, we need to update constructor to grant minter role to caller by default.
 
 ```rust
-// ::ink_lang_ir::Selector::new("MINTER".as_ref()).as_bytes()
-const MINTER: RoleType = 0xfd9ab216;
+const MINTER: RoleType = brush::blake2b_256_as_u32!("MINTER");
```

Comment:
Let's place a comment on why we're using hash on "MINTER". There is a good idea behind it, but might be confusing for the reader.